shared/components/FormField/__tests__/FormField.spec.jsx.md

```diff
@@ -1 +1 @@
-// FormField is used by Textarea and all functionality is tested through it.
+// FormField is used by Textarea and Input, and all functionality is tested through them.
```

Comment:
Good catch! I poked around and noticed `Select` also uses FormField.

Reply:
Will update to:
> FormField is used by Input, Textarea and Select, and all functionality is tested through them.